test/it/live-connect.spec.js

```diff
@@ -189,7 +189,9 @@ describe('LiveConnect', function () {
     waitForRequests(probeLS() ? 1 : 2, server)
 
     const firstTrackingRequest = server.getTrackingRequests()[0]
-    if (!isSafari71() && !isIE9Or10()) {
+    if (isMobileSafariAfter14()) {
+      expect(firstTrackingRequest.query.refr).to.match(/http:\/\/schmoogle.com:3001.*/)
```

Comment:
What is actually matched here with .* here?

Reply:
browserstack ios 15 adds a `/` at the end. The regex could be `http:\/\/schmoogle.com:3001\/?`

Reply:
Do we need `?` if `/` is always added for `isMobileSafariAfter14()`?

Reply:
Then `isMobileSafariAfter14` is misleading. Should it rather be something like `isMobileSafari14OrNewer`?

Reply:
ok https://github.com/LiveIntent/live-connect/pull/49/commits/19a6a511a61e051d224677cba3ad8409802ea974